### PR TITLE
Added pokerus filter to breeding window

### LINKED ISSUE #2506: [SUGGESTION]  Add pokérus as a filter in hatchery
**Current Behavior**
It's hard to find which pokemon don't have pokérus.

**Improved Behavior**
Make it possible to filter pokemon by if they have pokérus or not.

**Reason for change**
It would make it easier to make sure all your pokemon have pokérus.

### PR DESCRIPTION
**Pull request by Jaaslet**
image

I did not add infected to the filter, as it should never show up in the hatchery.
I made the text search smaller, to not add a new line to the filter. The filter was getting too big. If we never want to add more filters, we might need to rethink how the filters should look... And then we can make the box bigger again.

Closes #2506

### CODE REVIEW

File: src/components/breedingModal.html [hunk 1]
```diff
@@ -94,6 +94,14 @@ <h5 class="modal-title" id="breedingModalLabel">Day Care</h5>
                                     <!-- /ko -->
                                 </select>
                             </div>
+                            <div class="form-group col-md-3 col-6">
+                                <label>Pokerus Status</label>
```
Comment (Aegyo) [inline src/components/breedingModal.html:98]:
```suggestion
                                <label>Pokérus Status</label>
```

**Review by Aegyo — CHANGES_REQUESTED** [review]
Minor text gripe, otherwise perfectly fine :+1:

**Comment by Aegyo** [comment]
Do we want a Pokérus sort option too?

**Comment by Jaaslet** [comment]
> Do we want a Pokérus sort option too?

No idea. Personally, i can't see a use case, when people have the filter, so my plan is to leave it out, until someone gives me a good reason

**Comment by Qwertypop04** [comment]
> No idea. Personally, i can't see a use case, when people have the filter, so my plan is to leave it out, until someone gives me a good reason

Would a sort for number of EVs be more useful, maybe?

**Comment by Jaaslet** [comment]
> Would a sort for number of EVs be more useful, maybe?

Yeah, that would make more sense. Yeah, i will add it tomorrow. A sort option does not take up any UI space, so might as well

**Comment by CorgiOnNeptune** [comment]
A Display Value for EVs would also be a useful addition.

**Comment by Jaaslet** [comment]
I have added sorting option and display now. So it should be ready for review again.

**Review by Aegyo — APPROVED** [review]
#2574 will need a change updating how evs are calculated for the display and sort option, but everything seems to work here :+1:

**Comment by Jaaslet** [comment]
> #2574 will need a change updating how evs are calculated for the display and sort option, but everything seems to work here 👍

I will then merge this one, so i can fix it in the other PR

**Pull request merged**